eng/pipelines/typespec-requirement.yml

```diff
@@ -1,22 +0,0 @@
-trigger: none
```

Comment:
Just for my learning: how the automation know to find and execute this file? Like, I know github looks inside `.github/workflows` by convention. Does something similar exist for GH/ADO integration, or do we have some config for that, or what?

Reply:
That is configured on the DevOps side and stored with the pipeline you setup.

Reply:
After this file is deleted, the DevOps pipeline will be kept for history (and maybe older branches), and eventually deleted